@xen-orchestra/web/src/components/host/HostPifSidePanel.vue

```diff
@@ -0,0 +1,459 @@
+        <UiCardTitle>{{ $t('properties') }}</UiCardTitle>
+        <div class="content">
+          <!-- MTU -->
+          <VtsCardRowKeyValue>
+            <template #key>
+              {{ $t('mtu') }}
+            </template>
+            <template #value>
+              {{ pif.mtu === -1 ? $t('none') : pif.mtu }}
+            </template>
+            <template v-if="pif.mtu !== -1" #addons>
+              <UiButtonIcon
+                v-tooltip="copied && $t('core.copied')"
+                :icon="faCopy"
+                size="medium"
+                accent="brand"
+                @click="copy(String(pif.mtu))"
+              />
+            </template>
+          </VtsCardRowKeyValue>
+          <!-- SPEED -->
+          <VtsCardRowKeyValue>
+            <template #key>
+              {{ $t('speed') }}
+            </template>
+            <template #value>
+              {{ speed }}
+            </template>
+          </VtsCardRowKeyValue>
+          <!-- NETWORK BLOCK DEVICE -->
+          <VtsCardRowKeyValue>
+            <template #key>
+              {{ $t('network-block-device') }}
+            </template>
+            <template #value>
+              {{ networkNbd }}
+            </template>
+            <template v-if="networkNbd" #addons>
+              <UiButtonIcon
+                v-tooltip="copied && $t('core.copied')"
+                :icon="faCopy"
+                size="medium"
+                accent="brand"
+                @click="copy(networkNbd)"
+              />
+            </template>
+          </VtsCardRowKeyValue>
+        </div>
+      </UiCard>
+    </template>
+  </UiPanel>
+</template>
+
+<script setup lang="ts">
+import { useNetworkStore } from '@/stores/xo-rest-api/network.store'
+import { usePifStore } from '@/stores/xo-rest-api/pif.store'
+import type { XoPif } from '@/types/xo/pif.type'
+import VtsCardRowKeyValue from '@core/components/card/VtsCardRowKeyValue.vue'
+import VtsConnectionStatus from '@core/components/connection-status/VtsConnectionStatus.vue'
+import VtsIcon from '@core/components/icon/VtsIcon.vue'
+import UiButton from '@core/components/ui/button/UiButton.vue'
+import UiButtonIcon from '@core/components/ui/button-icon/UiButtonIcon.vue'
+import UiCard from '@core/components/ui/card/UiCard.vue'
+import UiCardTitle from '@core/components/ui/card-title/UiCardTitle.vue'
+import UiPanel from '@core/components/ui/panel/UiPanel.vue'
+import UiTag from '@core/components/ui/tag/UiTag.vue'
+import UiTagsList from '@core/components/ui/tag/UiTagsList.vue'
+import { vTooltip } from '@core/directives/tooltip.directive'
+import { faCircle, faCopy, faEdit, faEllipsis, faStar, faTrash } from '@fortawesome/free-solid-svg-icons'
+import { useClipboard } from '@vueuse/core'
+import humanFormat from 'human-format'
+import { computed } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+const { pif } = defineProps<{
+  pif: XoPif
+}>()
+
+const { get } = useNetworkStore().subscribe()
```

Comment:
Since the code is complex, I think it would be better to name the network getter

```suggestion
const { get: getNetwork } = useNetworkStore().subscribe()
```